USING_MULTI_TENANCY.md

```diff
@@ -0,0 +1,53 @@
+# Multi-Tenancy
+
+Multi-tenancy allows a single `fhir-works-on-aws` stack to serve as multiple FHIR servers for different tenants.
+
+`fhir-works-on-aws` uses a pooled infrastructure model for multi-tenancy. This means that all tenants share the 
+same infrastructure (DynamoDB tables, S3 Buckets, Elasticsearch cluster, etc.), but the data 
+is logically partitioned to ensure that tenants are prevented from accessing another tenant’s resources.
+
+## Enabling multi-tenancy
+
+Use the `enableMultiTenancy` option when deploying the stack: 
+
+```bash
+serverless deploy --enableMultiTenancy true
+```
+
+**Note:** Updating an existing (single-tenant) stack to enable multi-tenancy is a breaking change. Multi-tenant 
```

Comment:
Is it worth mentioning once multi-tenancy is turned on, turning back to single tenant mold is also a breaking change?

Reply:
I added a small note about it.